presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -254,7 +255,8 @@ public BaseJdbcClient(
                 // skip unsupported column types
                 if (columnMapping.isPresent()) {
                     boolean nullable = (resultSet.getInt("NULLABLE") != columnNoNulls);
-                    columns.add(new JdbcColumnHandle(columnName, typeHandle, columnMapping.get().getType(), nullable));
+                    Optional<String> comment = Optional.ofNullable(emptyToNull(resultSet.getString("REMARKS")));
```

Comment:
Why `emptyToNull`?
(btw Postgres version doesn't have it)

In some databases you need to do something special to get remarks. (i don't remember details right now)

Reply:
> In some databases you need to do something special to get remarks.

Nvrm. 
if it works out of the box for Postgres, MySQL and sql server, there is nothing more to do.

Reply:
Forgot to mention the reason with comment, MySQL driver returns empty character even when it's null. I'll move this logic to MySQL connector and add some logic for SQLServer. It has special syntax instead of REMARKS column.

Reply:
> add some logic for SQLServer. It has special syntax instead of REMARKS column.

since the information is not immediately ready, can this cause a performance penalty?

Reply:
Tested with 1,000 tables with 10 columns (all columns have a comment). The time to get all column comment per table takes:
- 1st attempt 44s (44ms x 1000)
- 2nd~ attempt 12s (12ms x 1000)

Let me skip support for SQLServer just in case.